.envrc

```diff
@@ -1,3 +1,5 @@
+mkdir -p .direnv
```

Comment:
If we move this inside of the `use_flake` function we don't create it when we don't use nix. This might be useful if we create a esy setup in the future.

This is however not a blocker since I don't think there's a good workflow for esy and direnv currently.

What do you think @d4hines?

Reply:
Yeah, I think this is ok for now.